analyzers/tests/SonarAnalyzer.TestFramework/MetadataReferences/NuGetMetadataReference.cs

```diff
@@ -71,6 +72,7 @@ public static class NuGetMetadataReference
     public static References MicrosoftAspNetCoreRouting(string packageVersion) => Create("Microsoft.AspNetCore.Routing", packageVersion);
     public static References MicrosoftAspNetCoreMvcRazorRuntime(string packageVersion = "2.2.0") => Create("Microsoft.AspNetCore.Razor.Runtime", packageVersion);
     public static References MicrosoftAspNetCoreRoutingAbstractions(string packageVersion) => Create("Microsoft.AspNetCore.Routing.Abstractions", packageVersion);
+    public static References MicrosoftAspNetIdentity(string packageVersion = "3.0.0-rc1-final") => Create("Microsoft.AspNet.Identity", packageVersion);
```

Comment:
Every single version of this package is a pre-release :S
So I had to pin the last one instead of `LATEST`, because if you do `LATEST` there is a check to NOT download prerelease packages.

Reply:
Could you please add this comment to the code? It can avoid some questions in the future.